src/clusterfuzz/_internal/cron/cleanup.py

```diff
@@ -1070,6 +1072,11 @@ def update_component_labels(policy, testcase, issue):
   for filtered_component in filtered_components:
     issue.components.add(filtered_component)
 
+  # This is very specific to google_issue_tracker, so this attribute might not
+  # be available for other issue trackers.
+  if component_id and hasattr(issue, 'component_id'):
+    issue.component_id = component_id
```

Comment:
Have you checked that if this attribute is set, the we will make the right API call to buganizer to update it on the issue? Ideally there is a test for this already :)

Reply:
So that part is already tested in `google_issue_tracker_test.py`

Reply:
Awesome, thanks for checking.